tensorflow/buildspec-tf1.yml

```diff
@@ -40,24 +40,24 @@
         target: dockerd-entrypoint.py
 
   images:
-    BuildTensorflowCPUTrainPy3DockerImage:
```

Comment:
We need BuildTensorflowCPUTrainPy3DockerImage and the other py3(37) images in addition to BuildTensorflowCPUTrainPy36DockerImage and other py36 images